docs/index.rst

```diff
@@ -733,25 +733,27 @@ solution, Confluent Replicator is also configured with security.
 
    .. figure:: images/replicator_topic_info.png
 
+5. Notice that because |crep| default is ``topic.config.sync=true``, then the replicated
```

Comment:
I had to go look up the `topic.config.sync` docs to be certain I undetstood what it does.  The name is decently self-explanatory but I wanted to be sure..  For you consideration, a link or quick text of what this setting accomplishes might help a future reader.

Reply:
Good suggestion, done